tests/monitors/mongodb/mongodb_test.py

```diff
@@ -52,9 +60,9 @@ def test_mongo_enhanced_metrics():
         assert wait_for(p(tcp_socket_open, host, 27017), 60), "service didn't start"
 
         with Agent.run(config) as agent:
-            assert wait_for(
-                p(has_datapoint, agent.fake_services, metric_name="gauge.collection.size"), 60
-            ), "Did not get datapoint from SendCollectionMetrics config"
-            assert wait_for(
-                p(has_datapoint, agent.fake_services, metric_name="counter.collection.commandsTime"), 60
-            ), "Did not get datapoint from SendCollectionTopMetrics config"
+            verify(
+                agent,
+                METADATA.metrics_by_group["collection"]
```

Comment:
Does `EXPECTED_DEFAULTS` not get received even with all enhance flags on? Or is there another reason you're ORing groups?

Reply:
`|` means to union all the groups together right?  I mean to test for everything coming through.

Reply:
What I really meant to say was all metrics minus the excluded ones:

```python
METADATA.all_metrics - {
    # These metrics only occur on MMAP storage engines.
    "gauge.backgroundFlushing.average_ms",
    "gauge.backgroundFlushing.last_ms",
    "counter.backgroundFlushing.flushes",
    # This one seems to be missing on newer mongo versions
    "gauge.extra_info.heap_usage_bytes",
}
```